puppet/modules/jenkins_job_builder/files/centos.org/pipelines/test/foremanNightlyTest.groovy

```diff
@@ -20,15 +20,15 @@ pipeline {
         }
         stage('Run Pipeline') {
             steps {
-                duffy_ssh("cd forklift && ansible-playbook pipelines/pipeline_katello_nightly.yml -e forklift_state=up", 'duffy_box', './')
+		duffy_ssh("cd forklift && ansible-playbook pipelines/bats_pipeline_foreman_nightly.yml -e 'forklift_state=up'", 'duffy_box', './')
```

Comment:
Tabs vs spaces. Also wondering if you intended this

Reply:
Negative, it's because of something wierd I have locally, apologies. I'll fix the other issue as well.